libs/sync/event.go

```diff
@@ -123,19 +140,26 @@ type EventSyncComplete struct {
 
 func (e *EventSyncComplete) String() string {
 	if e.Seq == 0 {
+		if e.DryRun {
+			return "[DRY-RUN] Initial Sync Complete"
```

Comment:
Do we need to prefix every log message with [DRY-RUN]? 

If we do, we should probably just modify the logger to do so without updating every message.

However, I don't think we need that, we should output the messages as is. Taking inspiration from rsync: dry-run output looks the same as normal:
```

/tmp % rsync -av --dry-run abc tmp
Transfer starting: 2 files
created directory tmp
abc/hello

sent 115 bytes  received 32 bytes  58 bytes/sec
total size is 0  speedup is 0.00
```

Reply:
I've made the output less verbose in the follow-up commit: only one warning message about dry-run in the top of the output